src/main/frontend/mixins.cljs

```diff
@@ -99,7 +99,7 @@
      :did-mount (fn [state]
                   (attach-listeners state)
                   state)
-     :did-remount (fn [old-state new-state]
+     :will-remount (fn [old-state new-state]
```

Comment:
**non-blocking**: when renaming binding names, it requires a re-indent.

Reply:
I missed that. Handled.